include/tvm/runtime/ndarray.h

```diff
@@ -159,6 +159,16 @@ class NDArray {
    * \return The created NDArray view.
    */
   TVM_DLL static NDArray FromDLPack(DLManagedTensor* tensor);
+  /*!
+   * \brief Create a NDArray from a dltensor.
+   *
+   * This allows us to create a new NDArray from given dltensor.
+   *
+   * The memory is retained until the NDArray went out of scope.
+   * \param dl_tensor The DLTensor to copy data from.
+   * \return The created NDArray.
+   */
+  TVM_DLL static NDArray FromDLTensor(DLTensor* dl_tensor);
```

Comment:
This name suggest it returns a view of dl_tensor (FromDLPack and ToDLPack don't copy). CopyFromDLTensor might be a better name if is a copy